packages/core-components/src/lib/unsorted/viz/Value.svelte

```diff
@@ -86,79 +88,19 @@
 {:else if !error}
 	<PulseNumber value={formatValue(value, fmt)} />
 {:else}
-	<span class="error">
-		<span class="error-label">Error</span>
-		<span class="additional-info-icon">
-			<HelpCircleIcon height="18" width="18" verticalOffset="1" color="--grey-100" />
-		</span>
-		<span class="error-msg">{error}</span>
+	<span
+		class="group inline-flex gap-1 items-center relative cursor-help text-white font-sans text-sm bg-red-700 rounded-2xl pl-2 pr-[1px] mx-0.5"
+	>
+		<span class="inline pl-1">Error</span>
+		<Icon src={HelpCircle} class="w-6 h-6 text-gray-100 pb-0.5 pt-[1px]" />
```

Comment:
Seems like the error view is quite a bit bigger now, is that intentional?

Reply:
I built them side-by-side so I'm not sure where that would be coming from; is it the icon or the entire error view that you're seeing issues with?